source/docs/changelogs/2015/07.md

```diff
@@ -0,0 +1,34 @@
+---
+title: July Changelog
+description: A list of the most recent platform improvements and documentation.
+---
+##Platform Improvements
+
+####Single Sign-On (SSO) Release  
+We added two new features to help security conscious organizations work with our platform:
+
+[**SAML Login**](https://pantheon.io/docs/articles/organizations/saml-for-orgs/): For organizations with SAML compatible single sign-on solutions, Pantheon allows authentication of users to the Dashboard via your own login system, giving you central administrative control over employee access to the platform.
+
+**Secure Runtime Access**: Pantheon protects all access to runtime resources including Drush, WP-CLI, or backing services like MySQL, behind the security of an encrypted [SSH tunnel](https://pantheon.io/docs/articles/local/ssh-tunnels-for-secure-connections-to-pantheon-services/). This mode requires developers first establish a secure channel between their workstation and the Pantheon platform for all communication.
+
+####Workflows > Jobs with Upgraded Notifications
```

Comment:
This seems ambiguous:
Workflow is greater than jobs
or
Workflow moving to jobs